src/datastructures.apib

```diff
@@ -140,3 +140,121 @@
         + timeTracking.deleted
         + timeTracking.updated
         + user.deactivated
+
+## WorkOrderData (object)
++ customer (object, required)
+    + type (enum[string], required)
+        + Members
+            + contact
+            + company
+    + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
++ datetime: `2017-05-26T10:01:49+00:00` (string, required)
++ description (string, optional)
++ department_id: `cef01135-7e51-4f6f-a6eb-6e5e5a885ac7` (string, optional)
++ layout_id: `a05320f6-b3ef-4ff9-a287-2074a12fb3e9` (string, optional)
++ milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional)
++ event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional)
++ intervention_address (object, optional)
+    + One Of
+        + customer (object, required)
+            + type (enum[string], required)
+                + Members
+                    + contact
+                    + company
+            + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+        + address (Address)
```

Comment:
Feels strange to call it `intervention_address` and pass a customer. What was the reasoning behind this?

Reply:
There are two ways you can show that the intervention happened at a different address than the one of the customer:

1. it is the address of a second customer (f.e. an overlapping company needs to get the bill for work done on one of their sub-companies)
2. it is a hand-filled address (so not coupled to a customer)

Reply:
`Intervention location` might make more sense than `intervention_address` in that case? (still with a customer or an address)

Reply:
`intervention location` seems better to me.

Reply:
but a contact or a customer can have different addresses... so shouldn't we provide more detail that just the link? and what if the address changes over time? I think storing the address in all cases, makes more sense... Keeping a link to where the address comes from, might make sense

Reply:
Kept the `intervention_address` property but made the customer a property of the address, so we always have actual and non changing address fields but with an optional link to a customer entity.